build.gradle

```diff
@@ -1,13 +1,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.61'
+    ext.kotlin_version = '1.5.10'
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
+        classpath 'com.android.tools.build:gradle:4.1.3'
```

Comment:
7.0.2

Reply:
I'm getting an error that SO says is related this line
```
// gradle gradle/bintray/install.gradle
to apply plugin: 'com.github.dcendents.android-maven'
```

How about I update this after migrating to Maven?

Reply:
Why not. So you first do the PR and then rebase this or you want me to merge this and you'll update this in the next PR?

Reply:
Let's merge this one and I'll bump these two versions in the migration PR ✅